docs/service-mesh/03-02-istio-patch.md

```diff
@@ -3,29 +3,28 @@ title: Istio patch
 type: Details
 ---
 
-As a core component, Istio installs with every Kyma deployment by default. The installation consists of two steps:
+As a core component, Istio is installed with every Kyma Deployment by default. To run Kyma, this default Istio installation needs some changes. The [Istio Kyma patch](../../components/istio-kyma-patch/README.md) application patches the already existing Istio installation so that Kyma can use it. Considering the above, the installation includes the following steps:
```

Comment:
```suggestion
As a core component, Istio is installed with every Kyma deployment by default. To run Kyma, this default Istio installation needs some changes. The [Istio Kyma patch](../../components/istio-kyma-patch/README.md) application patches the already existing Istio installation so that Kyma can use it. Considering the above, the installation includes the following steps:
```

Reply:
The patch doesn't customize anything at this point, it's a job that checks the detected istio deployment against a series of checkpoints.